src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -748,7 +748,7 @@ MergeTreeIndexGranulePtr MergeTreeIndexFullText::createIndexGranule() const
     /// have an old index.
     /// TODO: remove this at the end of 2024.
```

Comment:
This [blog post](https://clickhouse.com/blog/clickhouse-search-with-inverted-indices) still uses inverted indexes. Removing the error message without updating the blog post might be a bit confusing. It was one of the first results when searching for "Full Text Search in ClickHouse" on Google.